react/components/UserAddress/index.tsx

```diff
@@ -1,55 +1,68 @@
-import PropTypes from 'prop-types'
 import React, { useEffect } from 'react'
-import { graphql } from 'react-apollo'
+import { useQuery } from 'react-apollo'
 import { useCssHandles } from 'vtex.css-handles'
 import ADDRESS_QUERY from 'vtex.store-resources/QueryAddress'
 
 import Container from '../Container'
 import AddressInfo from './AddressInfo'
 
-const CSS_HANDLES = ['userAddressContainer']
+const CSS_HANDLES = ['userAddressContainer'] as const
 
-const UserAddress = ({
+type Props = {
+  /** Define if address should be displayed in a single line or in blocks */
+  variation: 'inline' | 'bar'
+  /** Define if street should be displayed */
+  showStreet?: boolean
+  /** Define if city and state should be displayed */
+  showCityAndState?: boolean
+  /** Define if postal code should be displayed */
+  showPostalCode?: boolean
+  /** Define if prefix should be displayed */
+  showPrefix?: boolean
+  /** Define if component should render if address is empty */
+  showIfEmpty?: boolean
+}
+
+function UserAddress({
   variation,
-  addressQuery: addressQueryProp,
   showStreet = true,
   showCityAndState = false,
   showPostalCode = false,
   showPrefix = true,
   showIfEmpty = false,
-}) => {
-  const { orderForm } = addressQueryProp
+}: Props) {
   const handles = useCssHandles(CSS_HANDLES)
+  const { data: orderFormData, refetch } = useQuery(ADDRESS_QUERY, {
```

Comment:
Could you either type the `data` returned from `useQuery` or leave a TODO comment so that we don't forget to do it once this app is using the actual types from `vtex.product-context`?

The query this component is performing is this one: https://github.com/vtex-apps/store-resources/blob/master/react/queries/address.gql